workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -103,9 +159,9 @@ private void GenerateComponentGroups()
             }
         }
 
-        protected override void OnUpdate()
+        private void RunDispatchers()
```

Comment:
I think rename this. It isn't just dispatching it's basically dealing with all the lifecycle steps.